Xwt/Xwt/Key.cs

```diff
@@ -27,6 +27,7 @@
 
 namespace Xwt
 {
+	[Flags]
```

Comment:
Then again most shortcuts only use one key and muliple modifiers... I can change it if you want.